internal/cmd/pipeline/command_update.go

```diff
@@ -63,6 +63,19 @@ func (c *command) newUpdateCommand() *cobra.Command {
 }
 
 func (c *command) update(cmd *cobra.Command, args []string) error {
+	flags := []string{
+		"activation-privilege",
+		"description",
+		"name",
+		"ksql-cluster",
+		"secret",
+		"sql-file",
+		"update-schema-registry",
```

Comment:
alphabetize?

Reply:
oh man k does indeed go before n. 🤣

Reply:
If you use vim, highlight the whole code block, then type `:sort` 😎